meetings/2017-06-07.md

```diff
@@ -0,0 +1,115 @@
+# Node Foundation CTC Meeting 2017-06-07
+
+## Links
+ 
+* **Audio recording**: <https://www.youtube.com/watch?v=0gssTOvFsXg>
+* **GitHub Issue**: [CTC#140](https://github.com/nodejs/CTC/issues/140)
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1W690JFykwVpN_V8zCSs_6yDTnFqAgE3juRz5k28SMWg>
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1ZoLJImAGLJxivjLyCqzl7qXpjTOts1cKLwiR2XnzOLA>_
+
+## Present
+ 
+* Anna Henningsen @addaleax (CTC)
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Evan Lucas @evanlucas (CTC)
+* Franziska Hinkelmann @fhinkel (CTC)
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+* James M Snell @jasnell (CTC)
+* Josh Gavant @joshgav (observer/TSC)
+* Matteo Collina @mcollina (CTC)
+* Michael Dawson @mhdawson (CTC)
+* Brian White @mscdex (CTC)
+* Myles Borins @MylesBorins (CTC)
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Jenn Turner @renrutnnej (observer/Node.js Foundation)
+* Rod Vagg @rvagg (CTC)
+* Michaël Zasso @targos (CTC)
+* Trevor Norris @trevnorris (CTC)
+* Rich Trott @Trott (CTC)
+ 
+## Agenda
+ 
+* announcements
+* previous meeting review
+* issue review
+  * doc: note that EoL platforms are not supported [node#12672](https://github.com/nodejs/node/pull/12672)
+ 
+## Announcements
+ 
+* Congrats on 8.0.0 release!
+* LTS team working on a proposed charter for CTC review.
+ 
+## Previous Meeting Review
+ 
+* http: overridable keep-alive behavior of `Agent` [node#13005](https://github.com/nodejs/node/pull/13005)  _added to the agenda by @indutny_
+  * The PR was deemed uncontentious. Labeled ctc-review.
+* Invert dependency between core and readable-stream [node-eps#49](https://github.com/nodejs/node-eps/pull/49) _added to the agenda by @mcollina_
+  * Deferred until next meeting.
+ 
+## Minutes
+ 
+### doc: note that EoL platforms are not supported [#12672](https://github.com/nodejs/node/pull/12672)
+ 
+@mhdawson: suggestion to add to support policy that platforms are no longer supported once they reach end-of-life, even if Node.js hasn't changed versions.
+ 
+Question is what does support mean to begin with - is it just "best effort"? And what does it mean that it changes if the platform isn't supported?
+ 
+@mylesborins: A good example is Node 6 and Ubuntu 12.04. That version of Ubuntu isn't supported by Canonical, should we continue to support Node 6 on it?
+ 
+@mhdawson: If we tell people from the beginning of the semver-major that support for unsupported platforms will cease even within semver-major cycle, that would be okay.
+ 
```

Comment:
"Some of the discussion in the issue is that if we tell people from the beginning of the semver-major that support for unsupported platforms will cease even within semver-major cycle, that would be okay.